### ISSUE TITLE
SPA data in EAMxx has higher extrema then the data used in SCREAMv0

### ISSUE THREAD

**Issue by AaronDonahue** [E3SM-Project/scream#1470]
During the testing of #1419 , @brhillman noted that the maximum AER_G_SW was higher than expected.  This prompted an investigation of the SPA data files we are currently using for our EAMxx simulations.

The inconsistency may be from remapping the SPA data in SCREAMv0 simulations to a file suitable for EAMxx.  As far as I can tell the pipeline of getting from SCREAMv0 to EAMxx is as follows:

1. remap `spa_mixing_ratio_data.nc` to `SPA_file_unified_ne30.nc`
2. consolidate AER_X bands into a single variable, `SPA_file_unified_ne30.nc` -> `spa_file_unified_and_complete_ne30_scream.nc`
3. if needed, remap `SPA_file_unified_ne30.nc` to a different resolution (say ne4) and then redo step 2 on that file.

Looking at the max and min of the data in each step, it looks like the issue occurs in step 1.  The max of AER_SW_G for all bands in `spa_mixing_ratio_data.nc` is 1.0, while the max in `SPA_file_unified_ne30.nc` is 1.00757.  

Now an important points, `spa_mixing_ratio_data.nc` is defined on a lat/lon mesh, not an SE mesh.  So step 1 is mapping lat/lon to SE.  Could it  be that the remapping algorithm for this does some extrapolation somewhere?  My intuition says no, but if not then something else is happening.

**Comment by bartgol** [comment]
@AaronDonahue do you know what tools were used in each step? Perhaps even the exact command line?

**Comment by PeterCaldwell** [comment]
This seems wacky. the original spa fields came from an ne30 simulation (or an ne120 simulation, I don't remember which) so they must have been remapped to latlon, then mapped back to ne30, right? Why not just skip that step? Maybe @hassanbeydoun can clarify on Monday... It seems like there are opportunities to clean up the toolchain. Also, maybe we just want to redo the SPA v0 run(s) since I think E3SMv2 is officially released now?

**Comment by AaronDonahue** [comment]
Okay, this has been solved, thanks to some Slack conversation with @hassanbeydoun .  I was wrong in the workflow I listed above.  The file `SPA_file_unified_ne30.nc` is the RAW E3SMv2 output, which just so happens to have values outside of the bounds for AER_G_SW.  Hassan had to clip AER_G_SW when he remapped onto a lat/lon grid, which is where `spa_mixing_ratio_data.nc` came from.  He needed a lat/lon because that was the format expected by the interpolation routine in EAM-F90.  

So the missing step was that when I used the raw data file I also needed to clip the values of AER_G_SW to a max of 1.0.  I've done this and have an updated file for ne4 and ne30 resolutions.
